Frameworks/UIKit/UIScrollView.mm

```diff
@@ -217,7 +217,8 @@ - (BOOL)_isAnimating {
 
 - (void)_setupViewChangingHandler {
     __weak UIScrollView* weakself = self;
-    _viewChangingEventToken = _scrollViewer.ViewChanging(objcwinrt::callback([weakself] (const WF::IInspectable&, const Controls::ScrollViewerViewChangingEventArgs& arg) {
+    _viewChangingEventToken = _scrollViewer.ViewChanging(objcwinrt::callback([weakself](
+        const WF::IInspectable&, const Controls::ScrollViewerViewChangingEventArgs& arg) {
```

Comment:
so many formatting changes - makes it ~tough to find the fix.

---
In reply to: [112261921](https://github.com/Microsoft/WinObjC/pull/2565#discussion_r112261921) [](ancestors = 112261921)